users/management/commands/updateall.py

```diff
@@ -0,0 +1,12 @@
+from django.core.management.base import BaseCommand
+
+from utils.businesslogic import BusinessLogic
+
+# This should be safe to run
+
+
+class Command(BaseCommand):
+    help = "Updates all user info"
+
+    def handle(self, *args, **options):
+        BusinessLogic.update_all_users()
```

Comment:
This was something that should be ran with cron everyday, right? 

check emails app jobs folder, there are two examples on running cron jobs (ofelia docker and ofelia.ini will handle running them, just add the job :)

Reply:
No. This is to run command manually. It also needs to be run daily but that's another story.

Reply:
Should we add that now also?